config-connector/solutions/iam/kpt/member-iam/README.md

```diff
@@ -0,0 +1,84 @@
+Member IAM
+==================================================
+
+# NAME
+
+  member-iam
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create a service account in your desired project, and grant it a specific role (defaults to `compute.networkAdmin`) in the project.
+
+  You can also create any number of service accounts and grant them any number of roles by altering and applying the altered config YAMLs.
```

Comment:
I'm not sure we should explicitly tell the customer to do this. It makes it easy to lose track of what they've applied; the mapping between kpt package and deployment is lost.

Reply:
As we discussed offline, let's just leave this kind details at the comfort level of the solution dev and update them later if we receive any customer feedback.